lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/controller/GraphController.java

```diff
@@ -218,15 +234,17 @@ private void initListeners() {
 
         listen(Message.FILTERED, (controller, subject, args) -> {
             assert args.length == 1;
-            assert (args[0] instanceof Set<?>);
-
-            visibleSequences = (Set<Sequence>) args[0];
-            model.setVisible(visibleSequences);
-            diagram = new StackedMutationContainer(model.getBucketCache(),
-                    visibleSequences);
-            repaintNow = true;
-            repaint();
-        });
+            assert args[0] instanceof Set<?>;
+            // unfortunately java doesn't really let us typecheck generics :(
+                @SuppressWarnings("unchecked")
```

Comment:
Formatter does this. If I manually change it then the next time someone modifies this file and runs the formatter(as they should), it'll be reverted again.